controlplane/pkg/monitor_crossconnect_server/monitor_crossconnect_server.go

```diff
@@ -122,3 +130,27 @@ func (m *monitorCrossConnectServer) GetCrossConnect(crossconnectId string) (*cro
 func (m *monitorCrossConnectServer) SendCrossConnectEvent(event *crossconnect.CrossConnectEvent) {
 	m.crossConnectEventCh <- event
 }
+
+func StartNSMCrossConnectServer(model model.Model, address string) (error, *grpc.Server, MonitorCrossConnectServer) {
```

Comment:
Let's not promulgate this pattern.  

We are going to need to have multiple APIs listening on whatever the URL we have for the NSM.  This "Start${ServiceName}Server" pattern explicitly gives ownership of a Listener to a *single* service.

Reply:
Also... this MonitorCrossConnectServer was written to be generic, not nsmd specific, so we wouldn't be passing in nsmd.model.

Reply:
I've opened #506 to track this, but am going to go ahead and merge this patch.